Profile Folder/chrome/locale/vi-VN/properties/gsettings.properties

```diff
@@ -1,12 +1,12 @@
-geckiumSettings=[!!! Gèçƙïú₥ §èƭƭïñϱƨ !!!]
+geckiumSettings=Cài đặt Geckium
 
 addAnApp=Thêm ứng dụng
 
 themeHasNoDescription=Chủ đề này không có mô tả.
-builtInTheme=[!!! [ßÛÌ£T-ÌN] {{themeName}} !!!]
-showInFolder=[!!! §λôω ïñ ƒôℓδèř !!!]
-apply=[!!! Âƥƥℓ¥ !!!]
-yes=[!!! Ýèƨ !!!]
+builtInTheme=Chủ đề có sẵn
```

Comment:
`builtInTheme` needs a new translation.
`[BUILT-IN]` - It means that it's a theme that came with Firefox, for example: Alpenglow.
`{{themeName}}` - Is where the theme name will appear, without this, no name will be displayed in the theme's information/manager dialog.